roles/webserver/templates/nginx_unicorn.j2

```diff
@@ -18,7 +18,7 @@ server {
 
   include {{ app }}_ssl;
 
-  server_name www.{{ domain }};
+  server_name {{sub_domain}} {{ domain }};
```

Comment:
Really the domain variable should include the "www". So I wonder if rather than adding a `sub_domain` variable, we should rename `domain` to `hostname`, because hostname implies the whole domain?